alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
I don't think adding sputnik is a good choice here. I've included the code styles locally to allow them to be extended and modified easily, adding a dependency to do that greatly limits this.